MatrixKit/Controllers/MXKRoomViewController.m

```diff
@@ -2046,8 +2046,15 @@ - (void)dataSource:(MXKDataSource *)dataSource didCellChange:(id)changes
         NSArray *attachmentsWithThumbnail = self.roomDataSource.attachmentsWithThumbnail;
         [attachmentsViewer displayAttachments:attachmentsWithThumbnail focusOn:nil];
     }
-    
-    [self reloadBubblesTable:YES];
+
+    // If the user is scrolling while we reload the data for a new incoming message for example,
+    // there will be a jump in the table view display.
+    // Resetting the contentOffset after the reload fixes the issue.
+    CGPoint contentOffset = self.bubblesTableView.contentOffset;
+    if ([self reloadBubblesTable:YES] == NO)
```

Comment:
Could you please use an intermediate boolean value 'hasScrolledToTheBottom' to ease code review?